### PR TITLE
ObinsKit.ObinsKit version 1.1.5

### PR DESCRIPTION
**Pull request by headblockhead**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17267)

### CODE REVIEW

File: manifests/o/ObinsKit/ObinsKit/1.1.5/ObinsKit.ObinsKit.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: ObinsKit.ObinsKit
+PackageVersion: 1.1.5
+PackageLocale: en-US
+Publisher: Taicang Zhigengniao Information Technology Co. Ltd
```
Comment (OfficialEsco) [inline manifests/o/ObinsKit/ObinsKit/1.1.5/ObinsKit.ObinsKit.locale.en-US.yaml:6]:
```suggestion
Publisher: Taicang Zhigengniao Information Technology Co.,Ltd
```
Needs to match ARP

File: manifests/o/ObinsKit/ObinsKit/1.1.5/ObinsKit.ObinsKit.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: ObinsKit.ObinsKit
+PackageVersion: 1.1.5
+PackageLocale: en-US
+Publisher: Taicang Zhigengniao Information Technology Co. Ltd
+PackageName: ObinsKit
+License: Copyright © 2019 Taicang Zhigengniao Information Technology Co.,Ltd
+ShortDescription: ObinsKit installer
```
Comment (OfficialEsco) [inline manifests/o/ObinsKit/ObinsKit/1.1.5/ObinsKit.ObinsKit.locale.en-US.yaml:9]: Can you add a better description?

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/o/ObinsKit/ObinsKit/1.1.5/ObinsKit.ObinsKit.locale.en-US.yaml (hunk 2 above)
Comment (headblockhead) [reply manifests/o/ObinsKit/ObinsKit/1.1.5/ObinsKit.ObinsKit.locale.en-US.yaml]: Done!

File: manifests/o/ObinsKit/ObinsKit/1.1.5/ObinsKit.ObinsKit.locale.en-US.yaml (hunk 1 above)
Comment (headblockhead) [reply manifests/o/ObinsKit/ObinsKit/1.1.5/ObinsKit.ObinsKit.locale.en-US.yaml]: Done!

**Comment by OfficialEsco** [comment]
Close and re open the PR since the bot crashed

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**